src/draggable.ts

```diff
@@ -16,17 +19,29 @@ export function makeDraggable(
   const onPointerDown = (event: PointerEvent) => {
     if (event.button !== mouseButton) return
 
-    event.preventDefault()
-    event.stopPropagation()
+    // event.preventDefault() // <--- MODIFIED: REMOVED from here
+    // event.stopPropagation() // <--- MODIFIED: REMOVED from here
+
+    // NEW: Register the pointer
+    activePointers.set(event.pointerId, event)
+
+    // NEW: If this is a second finger, it's a pinch.
+    // Don't start a drag.
+    if (activePointers.size > 1) {
+      return
+    }
 
     let startX = event.clientX
     let startY = event.clientY
     let isDragging = false
     const touchStartTime = Date.now()
 
     const onPointerMove = (event: PointerEvent) => {
-      event.preventDefault()
-      event.stopPropagation()
```

Comment:
This can potentially (although unlikely) break dragging. Need to test Regions and drag-to-seek.